studio-docs/source/federated-graphs.mdx

```diff
@@ -0,0 +1,135 @@
+| `description` | `String` | Provides any additional helpful details about working with this subgraph or contacting its owner. This field also supports markdown links. |
+
+#### Known `@contact` limitations
+
+- To provide the `@contact` directive to Apollo, you must register your schema using the `--localSchemaFile` option of [`apollo service push`](./schema/cli-registration/#registering-a-non-federated-schema). If you use the `--endpoint` option instead, directives are **not** included in the registered schema.
+- Similarly, you cannot provide your `@contact` directive to Apollo via [schema reporting](./schema/schema-reporting/). This registration method also does not include directives.
+- If your subgraph doesn’t already have a `schema` object, you need to add one to use the `@contact` directive. The `schema` object cannot be empty, so you need to include at least one field in it (most commonly `query: Query`). If you do provide the `Query` type like this, it needs to include at least one field in your subgraph’s schema. If your subgraph’s `Query` type is empty, you need to add a dummy field for the SDL to be valid:
+
+    ```graphql:title=schema.graphql
+    @contact(...)
+    schema {
+      query: Query
+    }
+
+    extend type Query {
+      _dummyField: Boolean
+    }
+    ```
+
+### Viewing contact info
+
+After you register a subgraph schema with the `@contact` directive, the contact information is included in the metadata shown in Studio’s **Schema > SDL** tab:
+
+<img class="screenshot" src="./img/federated-graphs/subgraph-contact-info-on-sdl-closeup.jpg" width="424" />
+
+In the **Schema > Reference** tab, you’ll also see a contact card when you hover over the subgraph link of a type or field:
+
+<img class="screenshot" src="./img/federated-graphs/schema-reference-contact-directive-tooltip-closeup.jpg" width="388" />
+
+## Federation support in the Explorer
+
+The [Apollo Studio Explorer](./explorer/) provides enhanced support for working with federated graphs:
+- You can preview how your graph resolves data for an operation by inspecting the operation’s query plan in the Response pane.
+- While you’re building operations in the Explorer, you can see which subgraph each field of your operation originates in by enabling subgraph hints from the Explorer’s Settings tab.
+
+## Viewing composition errors
+
+Studio displays a **Federated** label next to the name of a federated graph. If a federated graph has composition errors, you can click the label to view the errors that must be resolved before composition can succeed.
+
+<img class="screenshot" src="./img/federated-graphs/federated-graph-label-composition-error-state.jpg" width="426" />
+
+<img class="screenshot" src="./img/federated-graphs/federated-graph-composition-error-modal.jpg" width="736" />
+
+[Learn more about composition errors](https://www.apollographql.com/docs/federation/errors/)
```

Comment:
This line gets lost below the screenshots, so let's move it _above_ the screenshots and put it in a blockquote (`>`)

Reply:
@StephenBarlow it looks kinda odd above the screenshots. But putting it in the quote def helps call attention to it. What do you think about putting it in the quote, but leaving it below the screenshots?

| above | below |
|---|---|
|![image](https://user-images.githubusercontent.com/1319791/113343574-dba2a400-92e4-11eb-9079-32fc479e26e1.png)|![image](https://user-images.githubusercontent.com/1319791/113343533-ca599780-92e4-11eb-99aa-df845a15b7ff.png)|